### DISCUSSION TITLE
[Bug] Helm Upgrade to replace the Docker Image with new Tag number.

### DISCUSSION THREAD

**Discussion by williamyao1982** [apache/skywalking#9290] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Helm Chart (apache/skywalking-kubernetes)

### What happened

The helm install first version to kubernetes is quite good. It will init-es to initial the database structure in ES. But when we setup the new image tag number with helm upgrade. The skywalking-es-init Job will also deploy with 2nd version of Helm chart. It will bring the error message and failed to deploy the new helm chart. 

### What you expected to happen

In your es-init.job.yaml, there is another enabler should be control like this is the second upgrade and no need to initail the ES.

### How to reproduce

Helm upgrade --install by second or more than second time to change the image tag number. 

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
This could be a good idea to enhance. But not a bug to track.

FYI @hanahmily WDYT? 
@williamyao1982 If you know how to do, we are welcome to see your pull request.

**Reply by williamyao1982** [reply]
In my option, there is two way to solve this issue. 
One is setup the annotations for this job with "helm.sh/hook": post-install and "helm.sh/hook-weight": "-1". But I guess it will impact if someone want to enable ES after first helm install.
Second one is setup the new value as .Value.oap.initEs = true by default. Use this one to control the job is effective or not. I will try this one.

**Reply by williamyao1982** [reply]
@wu-sheng How do I contribue the solution to your code to fix the Helm chart issue? Does any guideline?

**Reply by wu-sheng** [reply]
Submit a pull request like this, https://github.com/apache/skywalking-kubernetes/pull/87.
If you are not familiar with GitHub, there are many videos on the internet to introduce.

**Reply by williamyao1982** [reply]
I raised PR. https://github.com/apache/skywalking-kubernetes/pull/88
When you have time please review it. Thx.
